exercises/practice/pov/src/lib.rs

```diff
@@ -0,0 +1,38 @@
+#[derive(Clone, PartialEq)]
+pub struct Tree<T: Clone + PartialEq> {
+    _remove_this: std::marker::PhantomData<T>,
+}
+
+impl<T: Clone + PartialEq> Tree<T> {
+    #[must_use]
+    pub fn new(_label: T) -> Self {
+        todo!("Implement a function that creates a new Tree with given label");
+    }
+
+    #[must_use]
+    pub fn with_children(_label: T, _children: Vec<Self>) -> Self {
+        todo!("Implement a function that creates a new Tree with given label and Children");
```

Comment:
We prefer to avoid parameter names prefixed with underscore when possible. Please implement `Debug` on `Tree` and use `"{tree:?}"` in the format string. (goes for all functions here)